### DISCUSSION TITLE
What is the process going forward for corrections and improvements to the schemas?

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#639] [Q&A]
What is the process going forward for corrections and improvements to the schemas?

Is there a plan for a 1.1 or 2.0 schema?  Is there a timeline? If so, when will payers be required to switch over to the newer format? Are you collecting suggestions?

Who will be making the decisions about the changes?  Is there a comment period?

**Comment by dgolden** [comment]
Neil makes several excellent points. 

In addition, given the importance to the USA economy of medical services price transparency, and the expense of this endeavor, there should be a project retrospective no later than the one year point that solicits inputs from all interested parties.  

Per the Federal Register* the estimated one-time year one cost and hour burden for all issuers and TPAs for the In-Network Rate file (alone) was estimated to be $2,024,117,160.00.  

*https://www.govinfo.gov/content/pkg/FR-2020-11-12/pdf/2020-24591.pdf    See page 127, Table 12B

**Comment by shaselton-usds** [comment]
@nmayle @dgolden Thanks for asking.

Updates will continue to be made to the schema and you if you check the VERSION.md file, you can see the current schema is actually beyond v1.1 -- it is currently v1.3.1. These updates that are beyond v1.0 are optional at this point in time and are highly recommended as they contain optimizations. 

A major version release may (most probably will) contain breaking changes from v1.0 and thus most likely would be tied to the regulatory process to establish a new baseline.

**Reply by tdfow** [reply]
@shaselton-usds what is the best way to identify the pull request to align with the version update, as I am not seeing a cross-reference to identify what change was associated with a specific version easily.  Am I overlooking something?
